daemon/cmd/fqdn.go

```diff
@@ -231,14 +231,7 @@ func (d *Daemon) bootstrapFQDN(possibleEndpoints map[uint16]*endpoint.Endpoint,
 
 			metrics.FQDNGarbageCollectorCleanedTotal.Add(float64(len(namesToClean)))
 			_, err := d.dnsNameManager.ForceGenerateDNS(context.TODO(), namesToClean)
-			namesCount := len(namesToClean)
-			// Limit the amount of info level logging to some sane amount
-			if namesCount > 20 {
-				// namedsToClean is only used for logging after this so we can reslice it in place
-				namesToClean = namesToClean[:20]
-			}
-			log.WithField(logfields.Controller, dnsGCJobName).Infof(
-				"FQDN garbage collector work deleted %d name entries: %s", namesCount, strings.Join(namesToClean, ","))
+			metrics.FQDNGarbageCollectorDeletedEntriesTotal.Add(float64(len(namesToClean)))
```

Comment:
Wait a second, it turns out the metrics existed already, (line 232). So in reality we only need to remove the log lines? But it looks like @jrajahalme recently added this. Pinging @jrajahalme to double check if it's really useful to keep this message bring printed.

Reply:
Yes . I missed it , looks like line 232 does the same thing